samples/net/zperf/overlay-loopback.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_NET_DRIVERS=y
+CONFIG_NET_LOOPBACK=y
+CONFIG_NET_LOOPBACK_MTU=1100
+# Enable for testing TX only
+# CONFIG_NET_LOOPBACK_DROP_ALL_PACKETS=y
+
+CONFIG_NET_BUF_DATA_SIZE=1100
+# Optimized networking settings for performance
+CONFIG_NET_PKT_RX_COUNT=32
+CONFIG_NET_PKT_TX_COUNT=48
+CONFIG_NET_BUF_RX_COUNT=32
+CONFIG_NET_BUF_TX_COUNT=96
+CONFIG_HEAP_MEM_POOL_SIZE=200000
```

Comment:
Why heap? The sample does not use any heap at all.

Reply:
copy-paste, will remove.